apis/project_api/project_api/api/project.py

```diff
@@ -53,29 +53,22 @@ def get_available_sites():
 
     # Load the project config
     project = utils.get_project(version_str, project_id)
-
-    ## Hack: Have lat/lon in the project config file later
-    ensemble = sc.gm_data.Ensemble(project_id, config_ffp=project.ensemble_ffp)
+    # Read the project_id yaml which includes lat&lon information
+    with open(
+        Path(project.ensemble_ffp.rsplit("/", maxsplit=1)[0]) / f"{project_id}.yaml"
+    ) as f:
+        config = yaml.safe_load(f)
+    locations = config["project_parameters"]["locations"]
```

Comment:
I think this can be simplified by just getting project.config?
Then you can get the locations with
`locations = project.config["project_parameters"]["locations"]`

Reply:
Even better, thanks!